control-plane-operator/controllers/hostedcontrolplane/kas/kms/kms.go

```diff
@@ -0,0 +1,29 @@
+package kms
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apiserver/pkg/apis/config/v1"
+)
+
+type IKMSProvider interface {
+	GenerateKMSEncryptionConfig() (*v1.EncryptionConfiguration, error)
+
+	ApplyKMSConfig(podSpec *corev1.PodSpec) error
+}
+
+const (
+	KAS_CONTAINER_NAME          = "kube-apiserver"
```

Comment:
I'm curious why this is an all caps, C style const name?

Reply:
fixed